js_modules/dagster-ui/packages/ui-core/src/assets/insights/AssetCatalogActivityChart.tsx

```diff
@@ -16,53 +26,59 @@ export type ActivityChartData = {
   color: string;
 };
 
-export const ActivityChart = React.memo(({metrics}: {metrics: ActivityChartData}) => {
-  const {header, color, dataByDay, max} = metrics;
-  return (
-    <div className={styles.ActivityChartContainer}>
-      <Box flex={{direction: 'row', alignItems: 'center'}} padding={{bottom: 12}}>
-        <BodyLarge>{header}</BodyLarge>
-      </Box>
-      <div className={styles.ActivityChart}>
-        {dataByDay.map((dayData) => (
-          <ActivityChartRow
-            key={dayData.date}
-            date={dayData.date}
-            hourlyValues={dayData.hourlyValues}
-            max={max}
-            color={color}
-          />
-        ))}
-        <div className={styles.ActivityChartRow}>
-          <div />
-          <div className={styles.ActivityChartBottomLegend}>
-            <BodySmall>12AM</BodySmall>
-            <BodySmall>6AM</BodySmall>
-            <BodySmall>12PM</BodySmall>
-            <BodySmall>6PM</BodySmall>
-            <BodySmall>12AM</BodySmall>
+export const ActivityChart = React.memo(
+  ({metrics, unit}: {metrics: ActivityChartData; unit: string}) => {
+    const {header, color, dataByDay, max} = metrics;
+    return (
+      <div className={styles.ActivityChartContainer}>
+        <Box flex={{direction: 'row', alignItems: 'center'}} padding={{bottom: 12}}>
+          <BodyLarge>{header}</BodyLarge>
+        </Box>
+        <div className={styles.ActivityChart}>
+          {dataByDay.map((dayData) => (
+            <ActivityChartRow
+              key={dayData.date}
+              date={dayData.date}
+              hourlyValues={dayData.hourlyValues}
+              max={max}
+              color={color}
+              unit={unit}
+            />
+          ))}
+          <div className={styles.ActivityChartRow}>
+            <div />
+            <div className={styles.ActivityChartBottomLegend}>
+              <BodySmall>12AM</BodySmall>
+              <BodySmall>6AM</BodySmall>
+              <BodySmall>12PM</BodySmall>
+              <BodySmall>6PM</BodySmall>
+              <BodySmall>12AM</BodySmall>
```

Comment:
Might be nice to do this with awareness of the user's 12h/24h setting.

Reply:
Yeah I need to dig into this in a follow up